flink-python/pyflink/fn_execution/tests/test_process_mode_boot.py

```diff
@@ -114,7 +117,7 @@ def run_boot_py(self):
                 "--control_endpoint", "localhost:0000",
                 "--semi_persist_dir", self.tmp_dir]
 
-        return subprocess.call(args, stdout=sys.stdout, stderr=sys.stderr, env=self.env)
```

Comment:
Why change this?

Reply:
On Windows, the PyCharm IDE will replace the `sys.stdout` and `sys.stderr`, which does not have the attribute "fileno". If we specify the stdout and stderr here, an exception will thrown when running tests in the PyCharm IDE.